### ISSUE TITLE
Spoiler Threshold

### ISSUE THREAD

**Issue by roryeiffe** [roryeiffe/Spoiler-Block#10]
The extension pop-up needs to have an adjustable percentage threshold such that all websites that report back the percentage chance of spoilers that are more than the threshold are blocked and all that are less are not blocked.

**Comment by roryeiffe** [comment]
I'm currently running into an error when trying to 
`npm build`
Here is the error:
_Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' 'sha256-WiOM2d3Qp2pRQgkUUEOUL2m+pH8MH9rXfiWxtczSSys='". Either the 'unsafe-inline' keyword, a hash ('sha256-79OuphGfMUthbL1SDlqq/gGNOWLLcEZwwXlHKXXk0YI='), or a nonce ('nonce-...') is required to enable inline execution._
I will try to figure this out and report back to this issue with any updates.

**Comment by roryeiffe** [comment]
I found a fix to the build error in this [stack overflow article](https://stackoverflow.com/questions/16145522/chrome-showing-error-as-refused-to-execute-inline-script-because-of-content-sec/16153913). The fix was simple: adding a .env file in the root with the line "INLINE_RUNTIME_CHUNK=false" in it. 

I also completed the spoiler threshold interface. Right now, the user is able to either input their own threshold or use the up and down arrows to increment/decrement the threshold by 5.
